lib/src/interfaces/ComponentEvents.ts

```diff
@@ -20,7 +20,7 @@ export interface NavigationButtonPressedEvent extends ComponentEvent {
 }
 
 export interface ModalDismissedEvent extends ComponentEvent {
-  componentId: string;
+  componentName?: string;
```

Comment:
Why is componentName optional? Seems like we're always emitting it

Reply:
`dismissAllModals` do not emit `componentName`. I'm not sure whether it should return the name of the top modal component name? Let me know I can have a look.

Reply:
[You can get the componentName of the top modal like we get the id of the top modal](https://github.com/wix/react-native-navigation/blob/master/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/modal/ModalStack.java#L84)